tests/dummy/app/pods/application/template.hbs

```diff
@@ -1 +1,3 @@
-{{outlet}}
+<div class="dummy-body">
```

Comment:
If you put this in the application, then it won't be present when you port into the documentation site - is that the intention?